packages/app/components/stackbox/Stackbox.tsx

```diff
@@ -80,39 +94,85 @@ const balanceOptions = [
   { name: "Max", divider: BalanceDivider.MAX },
 ];
 
+const customParseAsTimestamp = createParser({
+  parse: (v) => {
+    const ms = parseInt(v);
+    if (Number.isNaN(ms)) {
+      return null;
+    }
+
+    const paramDate = new Date(ms);
```

Comment:
Not sure why but I see it as plural.

```suggestion
    const paramsDate = new Date(ms);
```